projects/demo/src/app/forms/layout-angular/layout-angular.html

```diff
@@ -51,15 +81,27 @@ <h1>Angular - {{layout}} - {{ labelSize ? 'grid' : 'no grid' }}</h1>
     </clr-radio-wrapper>
   </clr-radio-container>
   <clr-select-container>
-    <label>Select</label>
+    <label
+      >Select
+      <clr-signpost>
+        <cds-icon shape="info-standard" clrSignpostTrigger>Trigger</cds-icon>
+        <clr-signpost-content *clrIfOpen> Signpost content </clr-signpost-content>
+      </clr-signpost></label
+    >
```

Comment:
```suggestion
    <label>
      Select
      <clr-signpost>
        <cds-icon shape="info-standard" clrSignpostTrigger>Trigger</cds-icon>
        <clr-signpost-content *clrIfOpen>Signpost content</clr-signpost-content>
      </clr-signpost>
    </label>
```

unmangle tags and remove extraneous whitespace